rnndb/nvchipsets.xml

```diff
@@ -92,6 +92,7 @@
 
 	<value value="0x117" name="GM107"/>
 	<value value="0x118" name="GM108"/>
+	<value value="0x120" name="GM200"/>
```

Comment:
Well, you'd have to figure out which one is "newer", feature-wise, and put it second. Since I guess we have no good reasons to decide either way at this moment, I'd put it second on the off-chance that being date-newer also means it's also feature-newer (and because you don't get to do a global search-and-replace this way).